cmd/dist/dist.go

```diff
@@ -13,15 +13,28 @@ import (
 
 	"tailscale.com/release/dist"
 	"tailscale.com/release/dist/cli"
+	"tailscale.com/release/dist/synology"
 	"tailscale.com/release/dist/unixpkgs"
 )
 
+var synologyPackageCenter bool
+
 func getTargets() ([]dist.Target, error) {
-	return unixpkgs.Targets(), nil
+	var ret []dist.Target
+
+	ret = append(ret, unixpkgs.Targets()...)
+	ret = append(ret, synology.Targets(synologyPackageCenter)...)
```

Comment:
Is this supposed to be conditional on `synologyPackageCenter`?

Reply:
No, the bool is specifying which variant of the packages to build: standalone for sideloading, or the ones for distribution in the synology package center. The latter can request permission to use tuntap devices and such, whereas the sideloaded ones can't. Effectively, we're the only consumers of this flag, and we'll set it when generating packages for Synology to distribute, as opposed to the ones we put on pkgs.tailscale.com so people can sideload.

Reply:
Added a comment to expand on this at the call site.